src/mono/mono/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 project(mono)
 
+find_package(dncp REQUIRED)
```

Comment:
Is `dncp` required? If yes, should it get git-linked like `dnmd`?

Reply:
I think I can avoid the dncp dependency. I did some work in dnmd to remove the requirement. I'll try it out locally.